app/components/widgets/AlertBox.vue.ts

```diff
@@ -95,6 +98,8 @@ export default class AlertBox extends WidgetSettings<IAlertBoxData, AlertBoxServ
   editingName: string = null;
   languages: any[] = [];
 
+  facemaskEnabled = this.facemasksService.getEnabledStatus();
```

Comment:
If this doesn't change after init then it's fine as property initializer. Can you please confirm that's the case? Thanks.

Reply:
it does change